web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -266,11 +266,13 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
     _utcExtentDateTimePickerConfiguration: function(name) {
 
         var that = this;
+        var seleniumCls = name.replace(' ', '-');
```

Comment:
Are we going to try and distinguis selenium test support classes somehow e.g. prefix with selenium-